app/deepseg.cc

```diff
@@ -600,7 +610,12 @@ int main(int argc, char* argv[]) try {
 		}
 	}
 	// default green screen background (at capture true geometry)
-	cv::Mat bg = cv::Mat(capGeo.value().second, capGeo.value().first, CV_8UC3, cv::Scalar(0, 255, 0));
+	cv::Mat bg;
+	if ( crop_region.height == 0) {
+		bg = cv::Mat(capGeo.value().second, capGeo.value().first, CV_8UC3, cv::Scalar(0, 255, 0));
+	} else {
+		bg = cv::Mat(crop_region.width, crop_region.height, CV_8UC3, cv::Scalar(0, 255, 0));
+	}
```

Comment:
DRY:
```suggestion
	std::pair<int, int> bg_dim = !crop_region.height ?
		{ capGeo.value().second, capGeo.value().first } :
		{ crop_region.width, crop_region.height };
	cv::Mat bg = cv::Mat(bg_dim.second, bg_dim.first, CV_8UC3, cv::Scalar(0, 255, 0));
```

Reply:
There is a mistake here! I must check this!

Reply:
I hoped the compiler can resolve the initializer list, but you might need to make it explicit as `std::make_pair(a, b)`.

Reply:
I have tried to solve this, but there is no advantage compared to my first code, only more code.